### PR TITLE
Multiple templates in methods

### PR DESCRIPTION
**Pull request by varunagrawal**
Finished up a major TODO with multiple template support extended to class methods as well.

Now we can do this in the wrapper:

```cpp
class Test {
  template<T={SO3, SO4}, D={double, float}>
  void coolLieFunc(const T& g, const D& d);
}
```

### CODE REVIEW

**Comment by dellaert** [comment]
A->B on GTDynamics wrapper?

**Comment by varunagrawal** [comment]
Working on that in parallel.

**Review by dellaert — APPROVED** [review]
Need a unit test before landing. Not on A->B though so sit on this until B reached :-)

**Comment by varunagrawal** [comment]
This already has a unit test.

**Comment by varunagrawal** [comment]
Whoops, shot from the hip. It's still on my local machine.

**Comment by varunagrawal** [comment]
Gtdynamics A->B done :slightly_smiling_face:

File: tests/fixtures/class.i [hunk 1]
```diff
@@ -12,8 +12,8 @@ class Fun {
   template<T={string}>
   This dhamaal(double d, T t);
 
-  // template<T={string}, U={size_t}>
-  // This pret(double d, T t, U u);
+  template<T={string}, U={size_t}>
+  This pret(double d, T t, U u);
```
Comment (dellaert) [inline tests/fixtures/class.i:16]: Nice !!! 

BTW pret? dhamaal? probably cute but not very informative and even possibly confusing.

Comment (varunagrawal) [reply tests/fixtures/class.i:16]: Just `Fun` in dutch and hindi. I was running out of names.

Comment (varunagrawal) [reply tests/fixtures/class.i]: Updated the method names to be more descriptive. :slightly_smiling_face:

**Pull request merged**